import "alice/wonderlogs/protos/uniproxy_prepared.proto";

package NAlice.NWonderlogs;

message TUniproxyEvent {
    oneof Value {
        TUniproxyPrepared.TMegamindRequest MegamindRequest = 1;
        TUniproxyPrepared.TMegamindResponse MegamindResponse = 2;
    }
}
